src/plugins/tables/tables.js

```diff
@@ -173,7 +174,12 @@ $document.on( "draw.dt", selector, function( event, settings ) {
 	// Identify that the table has been updated
 	$elm.trigger( "wb-updated" + selector, [ settings ] );
 } );
-
+$document.on( "keypress", ".paginate_button", function( event ) {
+	var $elm = $( event.target );
+	if (event.keyCode == 13){
+		location = $elm.context.href;
```

Comment:
The ```location``` global are undefined in this context because of the closure.

Reply:
It's fixed.

.on( "keypress", function( evn ){
				if ( evn.keyCode === 13 ){
					window.location = evn.target.href;
				}
			} )